### PR TITLE
Unit test/markdown parser

### PR DESCRIPTION
**Pull request by ical10**
Reference for unit testing Twitter usernames and hashtags:

- https://help.twitter.com/en/managing-your-account/twitter-username-rules
- https://help.twitter.com/en/using-twitter/replies-not-showing-up-and-hashtag-problems#:~:text=If%20you%20write%20123%23sofun,in%20front%20of%20the%20%23%20symbol.

### CODE REVIEW

File: __tests__/markdown-parser.test.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  test("should cut words in hashtag if punctuation is present", () => {
+    expect(parseHashtag(inputHashtagWithPunctuation)).toMatch(outputHashtagWithPunctuation);
+  });
+});
+
+const inputUsername = "@SubsocialChain is a decentralised social finance platform.";
+const outputUsername =
+  "[@SubsocialChain](https://twitter.com/SubsocialChain) is a decentralised social finance platform.";
+
+const inputUsernameWithCharPrefix = "123@TwitterSupport";
```
Comment (olehmell) [inline __tests__/markdown-parser.test.ts:61]: Lets collect all constants to object with name like `mocks`

File: __tests__/markdown-parser.test.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    expect(parseUsername(inputUsernameWithCharPrefix)).toMatch(inputUsernameWithCharPrefix);
+  });
+});
+
+const inputCombinedString = inputHashtag + inputUsername;
+const outputCombinedString = outputHashtag + outputUsername;
+
+describe("Markdown parser", () => {
+  test("should return a defined value when given a non-empty string", () => {
+    expect(textToMarkdownParser(longTweet)).toBeDefined();
```
Comment (olehmell) [inline __tests__/markdown-parser.test.ts:86]: You have functions `parseHashtags` and `parseUsername`, but here you have `textToMarkdownParser`. Why do you not follow one style for all, like `parseTextToMarkdown`

**Review by olehmell — CHANGES_REQUESTED** [review]

File: __tests__/markdown-parser.test.ts (hunk 2 above)
Comment (ical10) [reply __tests__/markdown-parser.test.ts]: Applied in 5557504

File: __tests__/markdown-parser.test.ts (hunk 1 above)
Comment (ical10) [reply __tests__/markdown-parser.test.ts]: Applied in 5557504

**Review by olehmell — APPROVED** [review]

**Pull request merged**